releases/enterprise.go

```diff
@@ -0,0 +1,29 @@
+package releases
+
+import (
+	"fmt"
+)
+
+type EnterpriseOptions struct {
+	Enterprise bool
```

Comment:
```suggestion
```

I think we can just imply enterprise when the options are set, assuming it's expected as a pointer?

Reply:
I've changed the implementation to use a pointer, but I don't love how the code is now littered with `nil` checks (whereas before the empty values "just worked"). But if you're fine with that then let's keep it :)